webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -194,49 +194,49 @@ def update_response_overview_plot(
         if not (ensembles and responses and corr_param_resp["response"] in responses):
             raise PreventUpdate
         selected_response = corr_param_resp["response"]
-        plots = []
-        obs_plots: List[PlotModel] = []
-
         loaded_ensembles = [
             load_ensemble(parent, ensemble_id) for ensemble_id in ensembles
         ]
 
+        response_plots = []
+        obs_plots: List[PlotModel] = []
+
         for index, ensemble in enumerate(loaded_ensembles):
             response = ensemble.responses[selected_response]
 
-            x_axis = response.axis
+            response_x_axis = response.axis
 
-            if isinstance(x_axis, pd.Index) and x_axis.empty:
+            if isinstance(response_x_axis, pd.Index) and response_x_axis.empty:
                 continue
```

Comment:
@DanSava Do you know why we need this `if` above, e.g. when this is the case and what would be the `else` statement?

Reply:
It is needed to account for the cases where a response is empty (an empty dataframe or the index of the dataframe is not set correctly) this happens when data is corrupted in storage or there is no data output for that response.

>  what would be the else statement?

Not sure what you mean by this, but not all if statements require an `else` clause. In this case on the else branch, you would execute all the code below that is why we don't have any `else` here.